force-app/main/default/classes/SubmitCompensations_Schedule_Test.cls

```diff
@@ -0,0 +1,29 @@
+@isTest
+private class SubmitCompensations_Schedule_Test {
+ // Seconds Minutes Hours Day_of_month Month Day_of_week optional_year
+ public static String schedule = '0 21 10 * * ?';
+    
+	@testSetup
+    static void setup(){
+        List<Compensation__c> comp_list = new List<Compensation__c>();
+        Compensation__c comp1 = new Compensation__c();
+        comp1.RecordTypeId = '0123h000000H5smAAC';
+        comp1.Name = 'Test Comp1';
+        comp1.Contact__c = '0033h000001shV0AAI';
```

Comment:
you should create the contact for testing purposes and then, you create the compensation. so you can put here something like con.Id instead of hard coding it.

Reply:
Ok, thanks i will change that